flocker/acceptance/endtoend/test_dataset.py

```diff
@@ -102,9 +102,14 @@ def _get_package_source(self, default_version=None):
         ),
     )
     @skip_backend(
-        unsupported={backends.LOOPBACK},
-        reason="Does not maintain compute_instance_id across restarting "
-               "flocker (and didn't as of most recent release).")
+        unsupported={backends.LOOPBACK, backends.OPENSTACK},
+        reason=(
+            "Loopback backend does not maintain compute_instance_id across "
+            "restarting flocker (and didn't as of most recent release). "
+            "OpenStack backend (as of 1.15.0) cannot reliably identify "
+            "the compute node if it has floating IP addresses."
```

Comment:
I've created an issue to re-enable the tests after 1.16.0 which is blocked by the 1.16 release issue: 
 * https://clusterhq.atlassian.net/browse/FLOC-2966

I'll add a link in a followup branch.